README.md

```diff
@@ -18,7 +18,17 @@ injecting your own 802.11 frames, combining technics (VLAN hopping+ARP
 cache poisoning, VOIP decoding on WEP encrypted channel, ...),
 etc.
 
-See
+# Installation #
+
+Scapy works without any external Python modules on Linux and BSD like operating systems. On Windows, you need to install some mandatory dependencies as described in the documentation.
+
+To benefit from all Scapy features, you might want to install Python modules, such as matplotlib or pycrypto.
```

Comment:
There will be quite a lot of changes with the links to do in installation.rst when we'll have switched to cryptography, so I don't think we should do this now... I will do another PR once it's done